CMakeLists.txt

```diff
@@ -32,6 +32,9 @@ option(FIREBASE_INCLUDE_ADMOB "Include the AdMob library."
 option(FIREBASE_INCLUDE_ANALYTICS
        "Include the Google Analytics for Firebase library."
        ${FIREBASE_INCLUDE_LIBRARY_DEFAULT})
+option(FIREBASE_INCLUDE_APP_CHECK
```

Comment:
Do we need to change any build scripts so that this will be false for release builds?